go.mod

```diff
@@ -94,6 +94,7 @@ require (
 	github.com/form3tech-oss/jwt-go v3.2.5+incompatible // indirect
 	github.com/fsnotify/fsnotify v1.5.1 // indirect
 	github.com/gedex/inflector v0.0.0-20170307190818-16278e9db813 // indirect
+	github.com/getsentry/sentry-go v0.11.0 // indirect
```

Comment:
Is this up to date, and from latest Go 1.17? My local go mod changes a few things around, and this is direct, no?

Reply:
Good catch, ran go mod tidy. Should be all good now